### PR TITLE
Adds additional directional signs for mapping

### PR DESCRIPTION
**Pull request by ChilbroBaggins**
Adds a few additional directional signs for mapping;
HOP
Food
Bar
Jani
image

### CODE REVIEW

**Comment by Visne** [comment]
Maybe make it say "HoP" instead of "HOP".

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml [hunk 1]
```diff
@@ -9,6 +9,15 @@
 
 # Directional Station Guide Signs
 
+- type: entity
+  parent: BaseSign
+  id: SignDirectionalBar
+  name: bar sign
+  description: A direction sign, pointing out which way the Bar is.
```
Comment (Visne) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml:16]:
```suggestion
  description: A direction sign, pointing out which way the bar is.
```

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml [hunk 2]
```diff
@@ -79,7 +97,16 @@
...
   components:
   - type: Sprite
-    state: direction_gravity
+    state: direction_gravity    
+
+- type: entity
+  parent: BaseSign
+  id: SignDirectionalHop
+  name: hop sign
+  description: A direction sign, pointing out which way Head of Personel's office is.
```
Comment (Visne) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/signs.yml:106]:
```suggestion
  description: A direction sign, pointing out which way Head of Personnel's office is.
```

**Comment by ChilbroBaggins** [comment]
> Maybe make it say "HoP" instead of "HOP".

image

**Comment by Visne** [comment]
Looks nice.

**Comment by ChilbroBaggins** [comment]
Better?
image

**Comment by Visne** [comment]
Yeah, it's better

**Comment by ChilbroBaggins** [comment]
How do you feel about this?

I also thought "Books" but that's so reductive. DnD? haha
image

**Comment by Visne** [comment]
LBRY is understandable enough I think

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**